lib/dependencies.sh

```diff
@@ -124,8 +145,32 @@ npm_rebuild() {
     else
       echo "Installing any new modules (package.json)"
     fi
-    npm install --unsafe-perm --userconfig $build_dir/.npmrc 2>&1
+    npm install --production=$production --unsafe-perm --userconfig $build_dir/.npmrc 2>&1
   else
     echo "Skipping (no package.json)"
   fi
 }
+
+npm_prune_devdependencies() {
+  local build_dir=${1:-} 
+  local npm_version=$(npm --version)
+
+  if [ $NODE_ENV == "test" ]; then
```

Comment:
Same as the yarn comment above, but wanted to point it out for completeness sake.